roles/external_secrets_operator/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2025 VEXXHOST, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+- name: Deploy Helm chart
+  run_once: true
+  kubernetes.core.helm:
+    name: "{{ external_secrets_operator_helm_release_name }}"
+    chart_repo_url: "{{ external_secrets_operator_helm_repository_url }}"
+    chart_ref: "{{ external_secrets_operator_helm_chart_name }}"
+    chart_version: "{{ external_secrets_operator_helm_chart_version }}"
+    release_namespace: "{{ external_secrets_operator_helm_release_namespace }}"
+    create_namespace: true
+    kubeconfig: "{{ external_secrets_operator_helm_kubeconfig }}"
+    values: "{{ _external_secrets_operator_helm_values | combine(external_secrets_operator_helm_values, recursive=True) }}"
+
+- name: Wait for 'external-secrets' deployment to be available
+  run_once: true
+  kubernetes.core.k8s_info:
+    api_version: apps/v1
+    kind: Deployment
+    name: "{{ external_secrets_operator_helm_release_name }}"
+    namespace: "{{ external_secrets_operator_helm_release_namespace }}"
+    kubeconfig: "{{ external_secrets_operator_helm_kubeconfig }}"
+  retries: 18
+  delay: 5
+  register: result
+  until: >
+    result.resources[0].status.conditions is defined and
+    result.resources[0].status.conditions |
+    selectattr('type', 'equalto', 'Available') |
+    selectattr('status', 'equalto', 'True') | list | length > 0
```

Comment:
https://github.com/vexxhost/atmosphere/blob/ff50269732f1641329080f97734200563fd134b8/roles/neutron/tasks/main.yml#L63-L68

Check this out :)